### PR TITLE
Init configuration vs run

### PR DESCRIPTION
**Pull request by garettarrowood**
Change so configuration is always setup.

### CODE REVIEW

**Comment by garettarrowood** [comment]
cc/ @bklang

File: lib/ringflux/plugin.rb [hunk 1]
```diff
@@ -13,12 +13,12 @@ class Ringflux::Plugin < Adhearsion::Plugin
...
-  run :ringflux, after: :punchblock do
+  init :ringflux, after: :punchblock do
     InfluxDB::Logging.logger = logger
-    new.start
+    new.configure
   end
 
-  def start
+  def configure
     logger.info "Connecting to InfluxDB #{config.username}@#{config.host} with database #{config.database}"
```
Comment (bklang) [inline lib/ringflux/plugin.rb]: I'd like to also include the logging clarification here

**Review by bklang — CHANGES_REQUESTED** [review]

File: lib/ringflux/plugin.rb (hunk 1 above)
Comment (garettarrowood) [reply lib/ringflux/plugin.rb]: What would you like to add to the current logging?

Comment (bklang) [reply lib/ringflux/plugin.rb]: Change `Connecting to` to `Configuring` since no connection is made at this point

**Review by bklang — APPROVED** [review]

**Pull request merged**